lib/backend/dynamo/dynamodbbk.go

```diff
@@ -89,6 +89,8 @@ type Config struct {
 	// WriteTargetValue is the ratio of consumed write capacity to provisioned
 	// capacity. Required to be set if auto scaling is enabled.
 	WriteTargetValue float64 `json:"write_target_value,omitempty"`
+	// Tenant is value to be used as partition key.
```

Comment:
Disable the auto scaling as well, we are using on demand provisioning for all the tables.